src/Main.java

```diff
@@ -0,0 +1,400 @@
+            System.out.println("Health: " + dungeonBoss.health);
+            Thread.sleep(1000);
+            System.out.println("Attack: " + dungeonBoss.attack);
+
+        }
+        dungeonBoss.name = "Dungeon Boss";
+        dungeonBoss.attack = combinedAttack;
+        dungeonBoss.health = combinedHealth;
+
+
+
+
+
+       System.out.println("Welcome to dungeon run!");
+       Thread.sleep(2000);
+       System.out.println("In this game you will have to slay as many monsters as possible and stay alive");
+       Thread.sleep(3000);
+       System.out.println("What would you like you character's name to be?");
+       String userName = playerInput.nextLine();
+        player.name = userName;
+        System.out.println("Great Choice!");
+        Thread.sleep(2000);
+        System.out.println("Here are " + userName + "'s Stats: ");
+        Thread.sleep(1000);
+        System.out.println("Attack: " + player.attack);
+        Thread.sleep(1000);
+        System.out.println("Health: " + player.health);
+        Thread.sleep(1000);
+        System.out.println("Stamina: " + player.stamina);
+        Thread.sleep(1000);
+        System.out.println("(Besides your attack which gets capped at 10 your base stats cannot go over 20)");
+        Thread.sleep(4000);
+       System.out.println("Lets begin!");
+        Thread.sleep(2000);
+
+
+       System.out.println(userName + "'s first encounter is a " + randomMonster + "!");
+       Character monster;
+       if (randomMonster.equals ("Zombie")) {
+           monster = zombie;
+           Thread.sleep(2000);
+           System.out.println("Zombie Stats: ");
+           Thread.sleep(1000);
+           System.out.println("Attack: " + zombie.attack);
+           Thread.sleep(1000);
+           System.out.println("Health: " + zombie.health);
+       }
+       else if (randomMonster.equals ("Ghost")) {
+           monster = ghost;
+           Thread.sleep(2000);
+           System.out.println("Ghost Stats: ");
+           Thread.sleep(1000);
+           System.out.println("Attack: " + ghost.attack);
+           Thread.sleep(1000);
+           System.out.println("Health: " + ghost.health);
+       }
+       else {
+           monster = goblin;
+           Thread.sleep(2000);
+           System.out.println("Goblin Stats: ");
+           Thread.sleep(1000);
+           System.out.println("Attack: " + goblin.attack);
+           Thread.sleep(1000);
+           System.out.println("Health: " + goblin.health);
+       }
+
+
+       while (monster.health > 0 && player.health > 0 && player.stamina > 0) {
+           Thread.sleep(2000);
+           System.out.println("Would you like to Attack it, recover, or run away?");
+           Thread.sleep(1500);
+           System.out.println("Attacking drains 1 stamina from the user");
+           Thread.sleep(500);
+           System.out.println("and running away drains 2 stamina from the user");
+           String userInput = playerInput.nextLine();
```

Comment:
can be wrapped in a function. I'm going to stop commenting these out for now, but look out for more things that can be wrapped for readability